### DISCUSSION TITLE
Unsafe account detector

### DISCUSSION THREAD

**Discussion by firiz** [WeAreMahsaAmini/Core#9] [Ideas]
A bot and extension to flag Twitter/Instagram unsafe accounts.

**Comment by syavash** [comment]
Note: how to make sure it cannot be abused.

**Comment by omid** [comment]
I have such code for Twitter, but the problem is that the Twitter API is so limited, so actually it takes ages to check an account with thousands of followers, followings and ….

And it's funny that there is no paid version or anything of the API to increase the limit!

There are some ways to increase the limit:
- Open an account for “Academic Research” purposes, but we know it's not an academic research!
- Ask them to increase the limit, they said we will answer your message in a couple of days, but I still haven't got anything for months!
- Create many API keys to cover the limit, but it's illegal, and they will close them as soon as they find it
- Your idea?!

**Comment by mory91** [comment]
We can use crowdsourcing and make a browser plugin for people to report unsafe people, this doesn't need any API.
Later we can crawl these reported accounts (In a separate process - offline manner) to see if the reports were accurate (With analytics and/or AI).

**Comment by omid** [comment]
@mory91 read this thread: https://twitter.com/Hamed/status/1562783411177078784
